pages/[category].jsx

```diff
@@ -78,6 +74,7 @@ const Categorypage = ({ category }) => {
           {
             categoryArray.map((categoryValue, i) => {
               return(
+                // keyをcategoryValueにするとエラーでる
                 <li key={categoryValue} onClick={() => onClickOpen(categoryValue)}>
```

Comment:
keyをインデックスからcategoryValueに修正をしたのですが、以下のエラーがでてしまいます。

> react-dom.development.js?61bb:67 Warning: Encountered two children with the same key, `[object Object]`. Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.

Reply:
頑張って英語を読みましょう笑
オブジェクトを渡しているのもおかしいし、keyが重複しているらしいですよ